tests/geocoding_tests.py

```diff
@@ -48,50 +48,114 @@ def test_geocode_adresses_view_load(self):
         url = "/geocoder/geocoding/columns"
-        table_name = "no_table"
+        tableDto = models.TableDto(10001, "no_table")
 
-        data = {"tableName": table_name}
+        data = {"table": tableDto.to_json()}
         response = self.get_resp(url, json_=data)
 
-        message = "No columns found for table with name {0}".format(table_name)
+        message = "No columns found for table with name {0}".format(tableDto.name)
         assert message in response
 
-    # def test_add_lat_lon_columns(self):
-    #     table = db.session.query(SqlaTable).first()
-    #     database = db.session.query(Database).filter_by(id=table.database_id).first()
-    #     database.allow_dml = True
-    #     db.session.commit()
-    #
-    #     table_name = table.table_name
-    #     lat_column_name = "lat"
-    #     lon_column_name = "lon"
-    #
-    #     columns = reflection.Inspector.from_engine(db.engine).get_columns(table_name)
-    #     number_of_columns_before = len(columns)
-    #
-    #     geocoding.Geocoder()._add_lat_lon_columns(
-    #         table_name, lat_column_name, lon_column_name
-    #     )
-    #
-    #     columns = reflection.Inspector.from_engine(db.engine).get_columns(table_name)
-    #     number_of_columns_after = len(columns)
-    #     assert number_of_columns_after == number_of_columns_before + 2
-    #     column_names = [column["name"] for column in columns]
-    #     assert lon_column_name in column_names
-    #     assert lat_column_name in column_names
+    def test_does_valid_column_name_exist(self):
+        table = db.session.query(SqlaTable).first()
+        table_name = table.table_name
+        column_name = (
+            reflection.Inspector.from_engine(db.engine)
+            .get_columns(table_name)[0]
+            .get("name")
+        )
+
+        response = geocoding.Geocoder()._does_column_name_exist(table_name, column_name)
+        assert True is response
+
+    def test_does_invalid_column_name_exist(self):
+        table = db.session.query(SqlaTable).first()
+        table_name = table.table_name
+        column_name = "no_column"
+
+        response = geocoding.Geocoder()._does_column_name_exist(table_name, column_name)
+        assert False is response
+
+    def test_load_data_from_all_columns(self):
+        table_name = "birth_names"
+        columns = ["name", "gender"]
+
+        data = geocoding.Geocoder()._load_data_from_columns(table_name, columns)
+        assert ("Aaron", "boy") in data
+        assert ("Amy", "girl") in data
+
+    def test_load_data_from_columns_with_none(self):
+        table_name = "birth_names"
+        columns = ["name", None, "gender", None]
+
+        data = geocoding.Geocoder()._load_data_from_columns(table_name, columns)
+        assert ("Aaron", "boy") in data
+        assert ("Amy", "girl") in data
+
+    def test_add_lat_lon_columns(self):
+        database = db.session.query(Database).first()
+        database.allow_dml = True
```

Comment:
Again changing DB values without reversing, will this be persisted or flushed automatically?